src/@utils/schaclSchema.ts

```diff
@@ -0,0 +1,42 @@
+import { retrieveShaclSchema } from '@utils/aquarius'
+import { ShaclSchemaField } from '@context/MarketMetadata/_shaclType'
+
+import { capitalizeFirstLetter } from '@utils/textTransform'
+
+function getMinMax(valueField: string, field: ShaclSchemaField) {
```

Comment:
what is the point of this method? Seems over engineered as the only usage of this method further down in this file can be replaced with:

```
const { minLength, maxLength } = field
```